frontend/src/app/modules/shared/components/dialog/confirm-dialog/confirm-dialog.component.ts

```
@@ -0,0 +1,74 @@
+   * @param {ConfirmEventType} enum - Custom confirm event.
+   * @group Emits
+   */
+  @Output() onHide: EventEmitter<ConfirmEventType> = new EventEmitter<ConfirmEventType>();
+
+  constructor(private readonly configs: DynamicDialogConfig,
+              private readonly ref: DynamicDialogRef) {
+    this.confirmationOptions = this.configs.data;
+  }
+
+  accept() {
+    if (this.confirmationOptions && this.confirmationOptions.acceptEvent) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)